charts/jenkins/values.yaml

```diff
@@ -391,7 +391,7 @@ master:
   ingress:
     enabled: false
     # Override for the default paths that map requests to the backend
-    paths: []
+    # paths:
```

Comment:
I'd prefer to leave the empty list as default. That allows auto completion.

Reply:
Fixed :)